src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -258,7 +258,14 @@ public override void DefaultVisit(Symbol symbol)
             bool shouldSkipPartialDefinitionComments = false;
             if (symbol.IsPartialDefinition())
             {
-                if (symbol is MethodSymbol { PartialImplementationPart: MethodSymbol implementationPart })
+                Symbol? implementationPart = symbol switch
+                {
+                    MethodSymbol method => method.PartialImplementationPart,
+                    SourcePropertySymbol property => property.PartialImplementationPart,
+                    _ => null
+                };
+
+                if (implementationPart is not null)
```

Comment:
I wonder if this existing check is actually correct for partial symbols that can't skip implementation?

Reply:
I would be concerned about a scenario like this: ([SharpLab](https://sharplab.io/#v2:EYLgtghglgdgPgAQEwEYCwAoBBmABABwgCcAXKCAG12VwGFcBvTXF6gFlwFkAGACgEpmrJhlZiuAgNxCWAXxm4FAehW4APAGcArmEhEAngD5OAUxIALAPYATXAGNLukzBJql23cSMLCpclQQOTilMWSA) for analogous method scenario)

```cs
public partial class C {
    void M0()
    {
        _ = Prop; // on hover do we see the doc comment?
    }
    
    /// <summary>Property comment</summary>
    partial int Prop { get; } // no impl
}
```

Thankfully I think the `!_isForSingleSymbol` logic on line 288 (at time of writing this comment) ensures that Quick Info does show a doc comment here. We could add a test.